client/sites-dashboard/components/sites-ellipsis-menu.tsx

```diff
@@ -146,6 +146,7 @@ export const SitesEllipsisMenu = ( {
 } ) => {
 	const dispatch = useDispatch();
 	const { __ } = useI18n();
+	const isNotAtomicJetpack = site.jetpack && ! site?.is_wpcom_atomic;
```

Comment:
Since this check is used in two different locations might be better to add it as a function in `utils.ts` then call that function?

Reply:
Agreed, I extracted this to `utils.ts`.

I also thought about reusing a method from state selectors, e.g. `isJetpackSitePred`, but it expects different types and is built to support a few other use cases.